inst/qml/ExploratoryFactorAnalysis.qml

```diff
@@ -151,15 +151,20 @@ Form
 			title: qsTr("Base analysis on")
 			RadioButton
 			{
-				value: "correlation"
+				value: "cor"
 				label: qsTr("Correlation matrix")
 				checked: true
 			}
 			RadioButton
 			{
-				value: "covariance"
+				value: "cov"
 				label: qsTr("Covariance matrix")
 			}
+			RadioButton
+			{
+				value: "mixed"
+				label: qsTr("Polychoric or tetrachoric correlation matrix")
+			}
```

Comment:
If I understand this correctly, then psych automatically determines which variables are continuous, ordinal and dichotomous by using the `mixed` argument? Do you have any idea how that is done? I wonder if that is a "robust" method 
```suggestion
				label: qsTr("Polychoric/tetrachoric correlation matrix")
```

Reply:
Precisely - the `mixedCor` function was the most practical approach that I could think of. 

Quoting the Psych documentation:
>mixedCor finds Pearson correlations for the continous variables, polychorics for the polytomous items, tetrachorics for the dichotomous items, and the polyserial or biserial correlations for the various mixed variables.
>mixedCor is essentially a wrapper for cor, polychoric, tetrachoric, polydi and polyserial. It first identifies the types of variables, organizes them by type (continuous, polytomous, dichotomous), calls the appropriate correlation function, and then binds the resulting matrices together.

By default, if a variable has 8 levels or more, it is automatically considered "continuous" by the function - if you had items with an 8-point Likert, the `polychoric/tetrachoric` option would render the same result as the `correlation matrix` one. It's possible to change this "default" value, though it's not currently implemented in this PR (thought it might be "too much information all at once").